homeassistant/components/fireservicerota/binary_sensor.py

```diff
@@ -63,6 +66,7 @@ def is_on(self):
         else:
             self._state = False
 
+        self._client.on_duty = self._state
```

Comment:
The binary sensor should not be responsible for keeping the client up to date. Instead the client should subscribe to the coordinator if it wants that data.

A user might choose to not include this entity.

Reply:
Have to think how to solve this, will have a look. Thank you.

Reply:
Fixed